conans/util/files.py

```diff
@@ -188,6 +188,10 @@ def save(path, content, only_if_modified=False, encoding="utf-8"):
     """
     try:
         os.makedirs(os.path.dirname(path))
+    except OSError as error:
+        if error.errno == errno.EACCES:
+            raise OSError("Could not create the directory '{}' due to lack of permission."
+                          " Please, check your user write permission.".format(path))
```

Comment:
Directory name is ``os.path.dirname(path)``, not ``path``

Reply:
Done